RubberduckTests/Grammar/ResolverTests.cs

```diff
@@ -2210,6 +2210,27 @@ End Sub
             }
         }
 
+        [Category("Grammar")]
+        [Category("Resolver")]
+        [Test]
+        public void ObjectPrintExpr_IsReferenceToLocalVariable()
```

Comment:
Do we need another test for the case of 
```
xxx.Print foo, bar; baz : xxx.Print baz, bar; foo
```

It's not clear from the grammar file what happens if there's an instruction separator after a `Print`, which ic a legal thing. See my other comment RE: grammar.

Reply:
That should work just fine as the colon is not a formatting character in the output list. I will add a test nonetheless.